### ISSUE TITLE
ExcelNPOIStorage Not Trimming

### ISSUE THREAD

**Issue by michaelakin** [MarcosMeli/FileHelpers#206]
I am trying to use the ExcelNPOIStorage with a class that has fields marked with         [FieldTrim(TrimMode.Both)] and it does not seem to be trimming.  I also tried the following settings:
        [FieldTrim(TrimMode.Both," ")]
        [FieldTrim(TrimMode.Right)]

**Comment by MarcosMeli** [comment]
@michaelakin You are right, the Excel feature don't is using some attributes, we need to do a full review of the attributes that can be used for Excel Storage :disappointed:
